### DISCUSSION TITLE
How to automatically generate titles with masks | 如何在用面具的情况下自动生成标题

### DISCUSSION THREAD

**Discussion by hexsix** [ChatGPTNextWeb/NextChat#3757] [Q&A]
Using a mask will disable the automatic title generation, and only the mask name will be displayed.

用面具之后自动生成标题就不起作用了，只会显示面具的名字。

**Comment by H0llyW00dzZ** [comment]
Depends of language I think
in english it work if you rename this `Kubernetes Journey` (as Example) to `New Conversation`

![image](https://github.com/ChatGPTNextWeb/ChatGPT-Next-Web/assets/17626300/710e2340-279f-4833-8273-ad280283ba9e)

![image](https://github.com/ChatGPTNextWeb/ChatGPT-Next-Web/assets/17626300/a6de982c-5581-43c8-bf94-26f7c1954f14)

**Reply by hexsix** [reply]
Language independent on my test. It depends on whether the current topic field value is the default one.

If I change the name of the mask to `新的聊天` (`New Conversation` in chinese), it will automatically generate a title after the second round of dialogue.

This is a way, but modifying the topic field to the default value is not what I want. I hope to automatically generate titles when using masks and the mask name is not `New Conversation`.
